sonic-chassisd/scripts/chassisd

```diff
@@ -69,6 +69,10 @@ CHASSIS_MIDPLANE_INFO_ACCESS_FIELD = 'access'
 CHASSIS_MODULE_HOSTNAME_TABLE = 'CHASSIS_MODULE_TABLE'
 CHASSIS_MODULE_INFO_HOSTNAME_FIELD = 'hostname'
 
+CHASSIS_MODULE_REBOOT_INFO_TABLE = 'CHASSIS_MODULE_REBOOT_INFO_TABLE'
+CHASSIS_MODULE_REBOOT_TIMESTAMP_FIELD = 'timestamp'
+REBOOT_SYSTEM_UP_WARNING_TIMEOUT = 180 
```

Comment:
How did we arrive at a value of 180 seconds for linecard reboot warning timeout? Was there an exercise done for calculating reboot time with expected and unexpected reboots? If yes, can you provide the values in both scenarios?

Reply:
@mlok-nokia can we add a linecard_reboot_timeout flag under platform dir and read from there? So that each vendor can define their platform specific timeout value.. 

We can keep default value to 180secs (if the flag is not defined under platform)?